src/mbgl/map/live_tile.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef MBGL_MAP_LIVE_TILE
+#define MBGL_MAP_LIVE_TILE
+
+#include <map>
+
+#include <mbgl/map/geometry_tile.hpp>
+
+namespace mbgl {
+
+class LiveTileFeature : public GeometryTileFeature, private util::noncopyable {
+public:
+    LiveTileFeature(FeatureType, GeometryCollection, std::map<std::string, std::string> = {{}});
```

Comment:
Please name the final parameter here; it's not obvious from the type what it is.